com.unity.render-pipelines.universal/Editor/ShaderGUI/Shaders/LitShader.cs

```diff
@@ -144,6 +144,20 @@ public override void AssignNewShaderToMaterial(Material material, Shader oldShad
                     material.SetTexture("_MetallicSpecGlossMap", texture);
             }
 
+            if (material.HasProperty("_ZClip"))
+            {
+                if (SystemInfo.supportsDepthClip)
+                {
+                    material.SetFloat("_ZClip", 1);
+                    //material.EnableKeyword("_ZClipEnable");
```

Comment:
I was looking at a few different ways to support the fallback, and I just left the code for using keywords in here for now for discussion purposes.

Reply:
At least I'm just fine with `_ZClip 1` and reference the property value .shader.
You could add an enum for the `1`. Maybe ZClip.On / ZClip.Off or Enable/Disable. Similar to Blend mode.